3/24 email (#44)
dolthub/weekly-updates: emails/2023-03-24.md

This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 

> With [HBO's Succession](https://www.hbo.com/succession) entering it's final season, a running joke in our office is that my son will be the next CEO of DoltHub. Once he came in and I asked him what he would do if [Zach](https://www.dolthub.com/team#zach) was watching YouTube all day at work. He said, "Fired". All I needed to hear.

Reviewer: ```suggestion
With [HBO's Succession](https://www.hbo.com/succession) entering its final season, a running joke in our office is that my son will be the next CEO of DoltHub. Once he came in and I asked him what he would do if [Zach](https://www.dolthub.com/team#zach) was watching YouTube all day at work. He said, "Fired". All I needed to hear.
```

### Hosted Dolt

> [Hosted Dolt](https://hosted.doltdb.com/) is quickly becoming the easiest way to run Dolt. We recently released improvements to the built in workbench. Clone a Hosted, better integration with DoltHub, and replication are all coming soon.

Reviewer: ```suggestion
[Hosted Dolt](https://hosted.doltdb.com/) is quickly becoming the easiest way to run Dolt. We recently released improvements to the built-in workbench. Clone a Hosted, better integration with DoltHub, and replication are all coming soon.
```

[![Hosted Dolt](../images/hosted-logo.png)](https://hosted.doltdb.com/)

Taylor wrote a [DoltHub vs Hosted Dolt blog](https://www.dolthub.com/blog/2023-03-17-dolthub-vs-hosted-workbench/) that details how we think about the workbench features of both. Our [new Pricing page](https://www.dolthub.com/compare) does the best job comparing all of our products if you or a friend is wondering which Dolt is right for you.

I've had a couple people reach out to me about a free trial instance of Hosted. If you want to try it but can't afford the $50/month, just reply to this email and say so and I'll set your account on "Do Not Collect".

### `dolt init --fun` and `dolt show`

> Thanks to [`solipsis-project`](https://github.com/solipsis-project). Both [`dolt init --fun`](https://github.com/dolthub/dolt/pull/5580) and [`dolt show`](https://github.com/dolthub/dolt/pull/5564) are brought to you by his contributions.

Reviewer 2: ```suggestion
Thanks to [`solipsis-project`](https://github.com/solipsis-project), both [`dolt init --fun`](https://github.com/dolthub/dolt/pull/5580) and [`dolt show`](https://github.com/dolthub/dolt/pull/5564) are brought to you by his contributions.
```

```bash
$ dolt init --fun
Successfully initialized dolt data repository.
$ dolt show
commit doltc1cgkfdf4ns2a1pd6j37pclivtbp (HEAD -> main) 
Author: timsehn <tim@dolthub.com>
Date:  Wed Mar 22 11:50:31 -0700 2023

        Іnitіalіze dаtа reposіtоry

```

> Note the commit hash. Maybe I'm a `d01t` but I couldn't be more happy about this silly feature.

Reviewer 2: ```suggestion
Note the commit hash. Maybe I'm a `d01t` but I couldn't be happier about this silly feature.
```
